### PR TITLE
Smith quantile docs

### PR DESCRIPTION
**Pull request by lukebrawleysmith**
Adding Quantile page under Statistics docs, and minor changes to metrics and Fact Tables docs.

### CODE REVIEW

File: docs/docs/statistics/quantile.mdx [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+Consider another universe where **all** customers are assigned to treatment. In the treatment universe, website latency for 99% of customers is no more than 139 ms. So the true effect at P99 is 139 ms - 145 ms = -6 ms.
+
+A quantile treatment test tries to estimate this difference. You interpret the interval above as “There is a 95% chance that the difference in P99 latencies across the groups is in (-9ms, -5ms)”. Put another way, if you were to run 100 experiments with different randomizations, roughly 95 of the intervals generated from those experiments would contain the value -6ms.
+
+### How do I run a quantile test in GrowthBook?
+
+Running a quantile test is just as easy as running a mean test. Currently, quantile testing is available only for [Fact Metrics](/app/fact-tables).
+
+1. Prerequisite: set up a “Fact Table” with your data you will use in your quantile test, if you do not already have one. If you want to compare, for example, median (p50) and mean effects, use a single Fact Table to create both metrics.
+2. On the Growthbook App select “Create Fact Table Metric”.
```
[the remark this answers is not in the document]
Comment (lukebrawleysmith) [reply docs/docs/statistics/quantile.mdx:37]: done

File: docs/docs/statistics/quantile.mdx [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+3. Select “Quantile” from “Type of Metric”.
+4. Toggle “Group by Experiment User before taking quantile” if you want to compare quantiles across variations at the user granularity, after summing row values at the user level. The default is at the event granularity. Please see the [FAQ](#faq) below for guidance regarding user vs event granularity.
+5. Pick your quantile level from the defaults (p50, p90, p95, p99) or use a custom value. Guidance describing the range of available values is in our [FAQ](#faq).
+6. Decide whether you want zeros to be included in the analysis (see [FAQ](#faq) below).
+7. Select your metric window as you would for a mean test.
+8. Submit!
+
+![User interface for quantile metrics](/images/statistics/quantile.png)
```
[the remark this answers is not in the document]
Comment (lukebrawleysmith) [reply docs/docs/statistics/quantile.mdx:45]: sounds good

File: docs/docs/statistics/quantile.mdx [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+
+## Growthbook implementation
+
+Here we describe technical details of our implementation.
+
+Growthbook implements the approach first introduced in [Deng, Knoblich and Yu (2018)](https://alexdeng.github.io/public/files/kdd2018-dm.pdf). This clever approach has two key advantages. First, it constructs valid confidence intervals for quantiles that uses only sample quantiles, rather than all of the data. This permits estimation using only a single pass through the data. Second, it provides quantile inference for clustered data. This is helpful when randomization occurs at the user level, but our metrics are measured at the session level (see [FAQ](https://www.notion.so/Quantile-Tests-173ec8d4b2264b858f87f2dbe779c285?pvs=21) below).
```
[the remark this answers is not in the document]
Comment (lukebrawleysmith) [reply docs/docs/statistics/quantile.mdx:51]: thanks for catching, I implemented the correct linking in other places, I missed this.

File: docs/docs/statistics/quantile.mdx [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+
+### How do I run a quantile test in GrowthBook?
+
+Running a quantile test is just as easy as running a mean test. Currently, quantile testing is available only for [Fact Metrics](/app/fact-tables).
+
+1. Prerequisite: set up a “Fact Table” with your data you will use in your quantile test, if you do not already have one. If you want to compare, for example, median (p50) and mean effects, use a single Fact Table to create both metrics.
+2. On the Growthbook App select “Create Fact Table Metric”.
+3. Select “Quantile” from “Type of Metric”.
+4. Toggle “Group by Experiment User before taking quantile” if you want to compare quantiles across variations at the user granularity, after summing row values at the user level. The default is at the event granularity. Please see the [FAQ](#faq) below for guidance regarding user vs event granularity.
```
[the remark this answers is not in the document]
Comment (lukebrawleysmith) [reply docs/docs/statistics/quantile.mdx:39]: done

File: docs/docs/statistics/quantile.mdx [hunk 5]
```diff
@@ -0,0 +1,80 @@
+---
+title: Quantile Testing
+description: Quantile Testing
+sidebar_label: Quantile Testing
+slug: quantile
+---
+
+# Quantile Testing
+
```
[the remark this answers is not in the document]
Comment (lukebrawleysmith) [reply docs/docs/statistics/quantile.mdx:9]: done

[the remark this answers is not in the document]
Comment (lukesonnet) [reply docs/docs/statistics/quantile.mdx:9]: Sorry can you remove Redshift and just say `Mixpanel or MySQL`? It works in Redshift....

File: docs/docs/experimentation-analysis/fact-tables.mdx [hunk 6]
```diff
@@ -111,6 +111,12 @@ For example, to create a **Session Duration** metric, you would do the following
 - Numerator: `SUM(duration)` from a `Sessions` fact table
 - Denominator: `COUNT(*)` from a `Sessions` fact table
 
+### Quantile Metrics
+
+Quantile Metrics are useful when you care about comparing variations at a quantile (e.g. latency at P99) rather than the mean. Learn more about when to use Quantile Metrics [here](/statistics/quantile).
+
+Constructing Quantile Metrics is identical to constructing [Mean Metrics](#mean-metrics).
```
Comment (lukesonnet) [inline docs/docs/experimentation-analysis/fact-tables.mdx:118]: Delete this line.

File: docs/docs/experimentation-analysis/metrics.mdx [hunk 7]
```diff
@@ -169,6 +169,10 @@ For example, if you want to look at the Average Order Value (AOV), what you're r
 1. `Orders per User` - selects the count of orders for each user
 2. `AOV` - selects total revenue per user and picks `Orders per User` as the denominator.
 
+##### Quantile Metrics
+
+[Quantile Metrics](/statistics/quantile) are available only as Fact Tables, as described [here](/app/fact-tables).
```
Comment (lukesonnet) [inline docs/docs/experimentation-analysis/metrics.mdx:174]: Try changing the link to `app/fact-tables#quantile-metrics`, it should take you right to the anchor.

**Review by lukesonnet — APPROVED** [review]
In general, LGTM.

Please change the image to the one I sent you, consider some of my latest comments, and consider simplifying the language about the delta method, or putting it in a separate section.

File: docs/docs/statistics/quantile.mdx [hunk 8]
```diff
@@ -0,0 +1,96 @@
...
+2. Select “Quantile” from “Type of Metric”.
+3. Toggle “Aggregate by Experiment User before taking quantile” if you want to compare quantiles across variations at the user granularity, after summing row values at the user level. The default is at the event granularity.
+4. Pick your quantile level from the defaults (p50, p90, p95, p99) or use a custom value. Guidance describing the range of available values is in our [FAQ](#faq).
+5. Decide whether you want zeros to be included in the analysis (see [FAQ](#faq) below).
+6. Select your metric window as you would for a mean test.
+7. Submit!
+
+![User interface for quantile metrics](/images/statistics/quantile.png)
+
+## Growthbook implementation
```
Comment (lukesonnet) [inline docs/docs/statistics/quantile.mdx:54]: Maybe move this whole section below the FAQ, TBH.

Comment (lukebrawleysmith) [reply docs/docs/statistics/quantile.mdx:54]: done, good call

File: docs/docs/experimentation-analysis/metrics.mdx (hunk 7 above)
Comment (lukebrawleysmith) [reply docs/docs/experimentation-analysis/metrics.mdx:174]: done

**Pull request merged**